### DISCUSSION TITLE
New container images are not published to hub.docker.com

### DISCUSSION THREAD

**Discussion by vitoyucepi** [renovatebot/renovate#26094] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.81.0-slim

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

The latest version of renovate/renovate container image hasn't been produced and published to https://hub.docker.com/r/renovate/renovate.

### Relevant debug logs

N/A

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment, marked as answer]
failed CI jobs, restarted jobs

**Reply by vitoyucepi** [reply]
Did it help for the 37.81.4 release?

**Reply by viceice** [reply]
the docker images are currently always delayed, because they're build on different repos which need to be updated by renovate app after main release. so releases can be deferred by several hours.

we're preparing to reduce the delay soon, so the docker images will be released minutes after the main GitHub releases.

**Reply by vitoyucepi** [reply]
If you are confident that everything is functioning properly and there are no issues, then I am content.
